tests/inspector/ckeditorinspector.js

```diff
@@ -26,9 +27,12 @@ describe( 'CKEditorInspector', () => {
 
 	afterEach( () => {
 		Logger.log.restore();
+		Logger.warn.restore();
```

Comment:
I'd change these two lines to `sinon.restore();` to be future-proof.

Reply:
It's deprecated https://github.com/sinonjs/sinon/issues/1390.

Reply:
These comments seem to issue the Sinon v1 and the sinon.restore( stub ) API. Now, the sinon namespace is also a sandbox, thus the `sandbox.restore()` works also for the default sandbox i.e. `sinon.restore()`.

See https://sinonjs.org/releases/latest/general-setup/.